src/System.Reflection/tests/MethodInfo/MethodInfo_PropertyTests.cs

```diff
@@ -71,38 +63,55 @@ public static void TestReturnParam2()
-            yield return new object[] { GetMethod(typeof(classA), "method1").IsGenericMethod, false };
-            yield return new object[] { GetMethod(typeof(classA), "method1").IsGenericMethodDefinition, false };
-            yield return new object[] { GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1").IsFinal, false };
-            yield return new object[] { GetMethod(typeof(classC), "virtualMethod").IsFinal, true };
-            yield return new object[] { GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1").IsConstructor, false};
-            yield return new object[] { GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1").IsAbstract, false};
-            yield return new object[] { GetMethod(typeof(classB), "abstractMethod").IsAbstract, true};
-            yield return new object[] { GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1").IsAssembly, false};
-            yield return new object[] { GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1").IsFamily, false};
-            yield return new object[] { GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1").IsFamilyAndAssembly, false };
-            yield return new object[] { GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1").IsFamilyOrAssembly, false };
-            yield return new object[] { GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1").ContainsGenericParameters, false};
-            yield return new object[] { GetMethod(typeof(classA), "GenericMethod").ContainsGenericParameters, true };
-            yield return new object[] { GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1").IsSpecialName, false};
-            yield return new object[] { GetMethod(typeof(classC), "abstractMethod").IsHideBySig, true};
-        }
-
-        //Verify Properties
-        [Theory]
-        [MemberData(nameof(TestPropertyData))]
-        public static void TestProperty(bool actual, bool expected)
+        //Verify DummyMethod1 properties
+        [Fact]
+        public static void TestProperty()
+        {
+            
+            MethodInfo dummyMethodInfo1 = GetMethod(typeof(MethodInfoPropertyTests), "DummyMethod1");
+
+            Assert.False(dummyMethodInfo1.IsVirtual);
+            Assert.True(dummyMethodInfo1.IsPublic);
+            Assert.False(dummyMethodInfo1.IsPrivate);
+            Assert.False(dummyMethodInfo1.IsStatic);
+            Assert.False(dummyMethodInfo1.IsFinal);
+            Assert.False(dummyMethodInfo1.IsConstructor);
+            Assert.False(dummyMethodInfo1.IsAbstract);
+            Assert.False(dummyMethodInfo1.IsAssembly);
+            Assert.False(dummyMethodInfo1.IsFamily);
+            Assert.False(dummyMethodInfo1.IsFamilyAndAssembly);
+            Assert.False(dummyMethodInfo1.IsFamilyOrAssembly);
+            Assert.False(dummyMethodInfo1.ContainsGenericParameters);
+            Assert.False(dummyMethodInfo1.IsSpecialName);
```

Comment:
Nit: it'd be helpful when there are lots of Assert.True and/or Assert.False calls in the same test to include messages for each (the second argument).  That way, it's easier to tell which one failed if one does fail.